migrations/20190919-initdb-builds.js

```diff
@@ -71,6 +71,9 @@ module.exports = {
                 },
                 stats: {
                     type: Sequelize.TEXT
+                },
+                subscribedConfigSha: {
```

Comment:
Migration files should not be upgraded but added like this file.
https://github.com/screwdriver-cd/data-schema/blob/master/migrations/20200204-add-column_templateId_to_builds.js